package fr.yvbro.rexa.exception

class RexaAuthentificationFailedException : RexaException("REXA_401", "Authentification failed")

